src/ceph-disk

```diff
@@ -1166,6 +1698,48 @@ def extract_parted_partition_numbers(partitions):
     numbers_as_strings = re.findall('^\d+', partitions, re.MULTILINE)
     return map(int, numbers_as_strings)
 
+
+# wait for udev event queue to clear
+def udev_settle():
+    time.sleep(1)
```

Comment:
I think we can remove this, I might have added a short delay so that there is some time between the partition create events reaching udev queue and the udev settle command flushing the queue. (assuming a second worth of time is enough for the kernel event to reach the udev on all platforms)